<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="loginPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="png files/pngFiles.qrc">:/loginPage/loginPage.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="loginBtt">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="png files/pngFiles.qrc">:/loginPage/loginBtt.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="signupBtt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="png files/pngFiles.qrc">:/loginPage/signupBtt.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameTxt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTxt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="png files/pngFiles.qrc"/>
 </resources>
 <connections/>
</ui>
